maps/away/noctis/noctis.dm

```diff
@@ -0,0 +1,96 @@
+
+/obj/effect/floor_decal/borderfloorgrey/corner2
+	icon_state = "borderfloorcorner2_white"
+
+/obj/effect/floor_decal/borderfloorgrey/full
+	icon_state = "borderfloorfull_white"
+
+/obj/effect/floor_decal/borderfloorgrey/cee
+	icon_state = "borderfloorcee_white"
+
+/obj/effect/paint/green_grey
+	color = "#8daf6a"
+
+/decl/prefab/ic_assembly/bluespace_radio
+	assembly_name = "bluespace radio" 					//Beware, dragons below
+	data = {"{'assembly':{'type':'type-c electronic assembly','name':'Bluespace Radio','detail_color':'#999875'},'components':\[{'type':'screen','name':'Device Owner'},{'type':'advanced integrated signaler','inputs':\[\[1,0,1357\],\[2,0,'BSNETGAMMA'\]\]},{'type':'microphone'},{'type':'text-to-speech circuit'},{'type':'text pad','name':'Assign New Owner'},{'type':'concatenator','inputs':\[\[2,0,': '\]\]},{'type':'and gate','name':'Microphone Check','inputs':\[\[1,0,0\]\]},{'type':'and gate','name':'Speaker Check','inputs':\[\[2,0,0\]\]},{'type':'toggle button','name':'Speaker Toggle'},{'type':'toggle button','name':'Microphone Toggle'},{'type':'slow ticker','inputs':\[\[1,0,1\]\]},{'type':'global positioning system'},{'type':'list constructor'},{'type':'len circuit','inputs':\[\[1,0,\[\]\]\]},{'type':'addition circuit','inputs':\[\[2,0,1\]\]},{'type':'join text circuit','inputs':\[\[1,0,\[\]\],\[2,0,', '\],\[4,0,5\]\]},{'type':'advanced integrated signaler','inputs':\[\[1,0,1357\],\[2,0,'GPSNETEPSILON'\]\]},{'type':'internal battery monitor'},{'type':'small screen','name':'Battery Charge'},{'type':'concatenator','inputs':\[\[2,0,'/'\],\[4,0,' ('\],\[6,0,'%)'\]\]},{'type':'number to string'},{'type':'number to string'},{'type':'number to string'}\],'wires':\[\[\[1,'I',1\],\[5,'O',1\]\],\[\[1,'A',1\],\[5,'A',1\]\],\[\[2,'I',3\],\[6,'O',1\]\],\[\[2,'O',1\],\[4,'I',1\]\],\[\[2,'O',1\],\[8,'I',1\]\],\[\[2,'A',1\],\[7,'A',2\]\],\[\[2,'A',3\],\[8,'A',1\]\],\[\[3,'O',1\],\[6,'I',1\]\],\[\[3,'O',2\],\[6,'I',3\]\],\[\[3,'A',1\],\[6,'A',1\]\],\[\[4,'A',1\],\[8,'A',2\]\],\[\[5,'O',1\],\[13,'I',4\]\],\[\[6,'O',1\],\[7,'I',2\]\],\[\[6,'A',2\],\[7,'A',1\]\],\[\[7,'I',1\],\[10,'O',1\]\],\[\[8,'I',2\],\[9,'O',1\]\],\[\[11,'A',1\],\[12,'A',1\]\],\[\[11,'A',1\],\[18,'A',1\]\],\[\[12,'O',1\],\[13,'I',1\]\],\[\[12,'O',2\],\[13,'I',2\]\],\[\[12,'O',3\],\[13,'I',3\]\],\[\[12,'A',2\],\[13,'A',1\]\],\[\[13,'O',1\],\[14,'I',1\]\],\[\[13,'O',1\],\[16,'I',1\]\],\[\[13,'A',2\],\[14,'A',1\]\],\[\[14,'O',1\],\[15,'I',1\]\],\[\[14,'A',2\],\[15,'A',1\]\],\[\[15,'O',1\],\[16,'I',4\]\],\[\[15,'A',2\],\[16,'A',1\]\],\[\[16,'O',1\],\[17,'I',3\]\],\[\[16,'A',2\],\[17,'A',1\]\],\[\[18,'O',1\],\[21,'I',1\]\],\[\[18,'O',2\],\[22,'I',1\]\],\[\[18,'O',3\],\[23,'I',1\]\],\[\[18,'A',2\],\[21,'A',1\]\],\[\[19,'I',1\],\[20,'O',1\]\],\[\[19,'A',1\],\[20,'A',2\]\],\[\[20,'I',1\],\[21,'O',1\]\],\[\[20,'I',3\],\[22,'O',1\]\],\[\[20,'I',5\],\[23,'O',1\]\],\[\[20,'A',1\],\[23,'A',2\]\],\[\[21,'A',2\],\[22,'A',1\]\],\[\[22,'A',2\],\[23,'A',1\]\]\]}"}
```

Comment:
I'm just going to assume you know what you're doing here

Reply:
I sure hope so. This is automatically generated by the in-game circuit exporter with the only alterations being escape characters.